### PR TITLE
@orta: Lesson four first draft.

### PR DESCRIPTION
**Pull request by ashfurrow**

### CODE REVIEW

File: Informed/Lesson Four/README.md [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+
+Nimble also uses the generic constraints we saw last time in [its operators](https://github.com/Quick/Nimble/blob/c3b755106ceb154c2512e4cee8ba5df9d3ad39af/Nimble/Matchers/Equal.swift#L102-L104):
+
+```swift
+public func ==<T: Equatable>(lhs: Expectation<T>, rhs: T?) {
+    lhs.to(equal(rhs))
+}
+```
+
+This means that the `==` operator applues only to things that conform to the `Equatable` protocol. Super cool!
```
Comment (orta) [inline Informed/Lesson Four/README.md]: applues

**Comment by orta** [comment]
Feels good, might end up a bit short time-wise ( maybe 30-45m) but :+1:

**Comment by ashfurrow** [comment]
Yeah, I agree. It's heavy on thinking, not on content. Planning on taking it slow and asking questions of our students. Correct the typo.

**Comment by orta** [comment]
word.

**Pull request merged**